contracts/payloads/IGP98/PayloadIGP98.sol

```diff
@@ -0,0 +1,293 @@
+pragma solidity ^0.8.21;
+pragma experimental ABIEncoderV2;
+
+import {BigMathMinified} from "../libraries/bigMathMinified.sol";
+import {LiquidityCalcs} from "../libraries/liquidityCalcs.sol";
+import {LiquiditySlotsLink} from "../libraries/liquiditySlotsLink.sol";
+
+import {IGovernorBravo} from "../common/interfaces/IGovernorBravo.sol";
+import {ITimelock} from "../common/interfaces/ITimelock.sol";
+
+import {IFluidLiquidityAdmin, AdminModuleStructs as FluidLiquidityAdminStructs} from "../common/interfaces/IFluidLiquidity.sol";
+import {IFluidReserveContract} from "../common/interfaces/IFluidReserveContract.sol";
+
+import {IFluidVaultFactory} from "../common/interfaces/IFluidVaultFactory.sol";
+import {IFluidDexFactory} from "../common/interfaces/IFluidDexFactory.sol";
+
+import {IFluidDex, IFluidAdminDex, IFluidDexResolver} from "../common/interfaces/IFluidDex.sol";
+
+import {IFluidVault, IFluidVaultT1} from "../common/interfaces/IFluidVault.sol";
+
+import {IFTokenAdmin, ILendingRewards} from "../common/interfaces/IFToken.sol";
+
+import {ISmartLendingAdmin} from "../common/interfaces/ISmartLending.sol";
+import {ISmartLendingFactory} from "../common/interfaces/ISmartLendingFactory.sol";
+import {IFluidLendingFactory} from "../common/interfaces/IFluidLendingFactory.sol";
+
+import {ICodeReader} from "../common/interfaces/ICodeReader.sol";
+
+import {IDSAV2} from "../common/interfaces/IDSA.sol";
+import {IERC20} from "../common/interfaces/IERC20.sol";
+import {IProxy} from "../common/interfaces/IProxy.sol";
+import {PayloadIGPConstants} from "../common/constants.sol";
+import {PayloadIGPHelpers} from "../common/helpers.sol";
+import {PayloadIGPMain} from "../common/main.sol";
+
+contract PayloadIGP98 is PayloadIGPMain {
+    uint256 public constant PROPOSAL_ID = 98;
+
+    function execute() public virtual override {
+        super.execute();
+
+        // Action 1: Update Range and center Price for USDC-USDT-CONCENTRATED DEX
+        action1();
+
+        // Action 2: Update Range and Center Price for WBTC-cbBTC DEX
+        action2();
+
+        // Action 3: Update Limits for sUSDe-USDT and USDe-USDT DEXes
+        action3();
+
+        // Action 4: Update Vault & DEX Auth for FLUID-ETH / ETH Vault and FLUID-ETH DEX
+        action4();
+    }
+
+    function verifyProposal() public view override {}
+
+    function _PROPOSAL_ID() internal view override returns (uint256) {
+        return PROPOSAL_ID;
+    }
+
+    /**
+     * |
+     * |     Proposal Payload Actions      |
+     * |__________________________________
+     */
+
+    // @notice Action 1: Update Range and center Price for USDC-USDT-CONCENTRATED DEX
+    function action1() internal isActionSkippable(1) {
+        {
+            address USDC_USDT_CONCENTRATED_DEX = getDexAddress(34);
```

Comment:
this should be on existing dex not on concentrated one, for concentrated we set up via team MS